packages/rolldown/tests/fixtures/builtin-plugin/glob-import/vite/fixture-a/index.ts.snap

```diff
@@ -83,7 +83,8 @@ const rawImportModule = {
 	"./modules/b.ts": () => import("./b3.js")
 };
 const excludeSelf = { "./sibling.ts": () => import("./sibling.js") };
-const excludeSelfRaw = { "./sibling.ts": () => import("./sibling2.js") };
+const excludeSelfRaw = { "./sibling.ts": () => import("./sibling3.js") };
+const customQueryString = { "./sibling.ts": () => import("./sibling2.js") };
```

Comment:
Can you ref related vite tests?

Reply:
The output seems not correct compare to https://github.com/rolldown/vite/blob/dd0fb27304716203e7d45492f87a4b247c9bc1a1/packages/vite/src/node/__tests__/plugins/importGlob/__snapshots__/fixture.spec.ts.snap, there is no `sibling3.js` and `sibling2.js`

Reply:
It seems caused by our test flow not same as vite, so ignore for now.

Reply:
Yeah, Vite test is for dev mode, so they just leave the query string as-is. We do full bundle, so the same file with different query string produces different chunks.